app/components/PolicyContainer.tsx

```diff
@@ -0,0 +1,102 @@
+import { useEffect, useState } from 'react';
+import logo from '../assets/full_logo.png';
+
+type ParsedLine = {
+  type:
+    | 'paragraph'
+    | 'heading1'
+    | 'heading2'
+    | 'heading3'
+    | 'listItem'
+    | 'break';
+  content: string;
+};
+
+type PolicyContainerProps = {
+  content: string;
+};
+
+export function PolicyContainer({ content }: PolicyContainerProps) {
```

Comment:
this component seems like it can render any markdown so I would not make the name policy specific